tests/spec/core/biblio-spec.js

```diff
@@ -209,3 +209,37 @@ describe("W3C — Bibliographic References", () => {
     expect(refs[0].textContent).toEqual("[dom]");
   });
 });
+
+it("makes sure the Informative references section has expected localization text", async () => {
+  const ops = {
+    config: makeBasicConfig(),
+    htmlAttrs: {
+      lang: "nl",
+    },
+    body: `<section class="informative" id="intro">[[DOM]]</section>`,
+  };
+  const doc = await makeRSDoc(ops);
+  const ref = doc.getElementById("references");
+  expect(doc.documentElement.lang).toEqual("nl");
+  expect(ref.querySelector("h2").textContent).toMatch("Referenties");
+  expect(ref.querySelector("section").querySelector("h3").textContent).toMatch(
+    "Informatieve referenties"
+  );
+});
+
+it("makes sure the normative references section has expected localization text", async () => {
+  const ops = {
+    config: makeBasicConfig(),
+    htmlAttrs: {
+      lang: "nl",
+    },
+    body: `<section>[[DOM]]</section>`,
+  };
+  const doc = await makeRSDoc(ops);
+  const ref = doc.getElementById("references");
+  expect(doc.documentElement.lang).toEqual("nl");
+  expect(ref.querySelector("h2").textContent).toMatch("Referenties");
+  expect(ref.querySelector("section").querySelector("h3").textContent).toMatch(
```

Comment:
Here we can be a bit more concise:
 
```suggestion
  const {textContent: h3TextContent} = doc.querySelector("#references h3");
  expect(h3TextContent.trim()).toBe("Normatieve referenties");
```